library.gradle

```diff
@@ -43,9 +51,21 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:4.6.1'
     testImplementation 'com.squareup.okhttp3:mockwebserver:4.2.0'
     testImplementation 'org.mockito:mockito-core:3.0.0'
-    testImplementation "com.android.billingclient:billing:$billingVersion"
+    testLatestDependenciesImplementation "com.android.billingclient:billing:$billingVersion"
+    testUnityIAPCompileOnly "com.android.billingclient:billing:$billingClient3Version"
     testImplementation 'io.mockk:mockk:1.10.0'
     testImplementation 'org.assertj:assertj-core:3.13.2'
 }
 
+def artifactId = project.property("POM_ARTIFACT_ID")
+def publishVariant = project.property("PUBLISH_VARIANT")
+if (publishVariant == "unityIAPRelease") {
+    project.ext.POM_ARTIFACT_ID = artifactId + "-unityiap"
+}
+println "$project.ext.POM_ARTIFACT_ID"
```

Comment:
Did you mean to leave this?

Reply:
https://github.com/RevenueCat/purchases-android/pull/522